build.gradle

```diff
@@ -52,6 +52,8 @@ distributions {
             into('lib') {
                 from configurations.runtime
                 from jar
+                exclude("**/common-sdk-${boomiCommonSdkVersion}.jar")
```

Comment:
instead of excluding the jars here, you should change the dependency scope to compileOnly - that is what we do for the other provided jars.  Based on the contents of the Atom's `lib/` directory, it looks like we need to change boomi-util to be compileOnly - the jars listed in this exclusion should *already* be excluded from the runtime configuration.

Reply:
I updated the build file and tested to make sure the dependency scopes compile, test, and build the appropriate distZip